src/components/evm/inputs/CurrencyInput.vue

```diff
@@ -984,7 +984,7 @@ export default defineComponent({
     --symbol-left: 28px;
     $this: &;
 
-    width: 300px;
+    max-width: 400px;
```

Comment:
```suggestion
```

Unless you have a specific reason why to set this maximum width I would suggest simply removing it because there are some cases you just want to have some control from the out side.

Check here the following two cases:

![Captura desde 2023-05-21 20-14-32](https://github.com/telosnetwork/telos-wallet/assets/4420760/48fce46a-e332-41d9-b069-d64d0e052b45)

![Captura desde 2023-05-21 20-15-14](https://github.com/telosnetwork/telos-wallet/assets/4420760/44d54c4b-67e6-4b5c-83cb-c53f37f38ea1)